tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -722,6 +732,7 @@ def __init__(self,
         use_quote_delim=self._use_quote_delim,
         na_value=self._na_value,
         select_cols=self._select_cols,
+        exclude_cols=self._exclude_cols,
```

Comment:
(Related to above comment of creating a new version of this op)

For forward compatibility reasons, here we'll need to do something along the lines of:

```
if compat.forward_compatible(2020, 06, 25):
  variant_tensor = gen_experimental_dataset_ops.csv_dataset_v2(...)
else:
  variant_tensor = gen_experimental_dataset_ops.csv_dataset(...)
```

We maintain a 3 week forward compat window on tensorflow. https://www.tensorflow.org/api_docs/python/tf/compat/forward_compatible